src/api/notification/policies/is-notification-owner.js

```diff
@@ -0,0 +1,15 @@
+
+module.exports = async (ctx) => {
+
+  const user = ctx.state.user;
+
+  ctx.request.query = ctx.request.query || {};
+  const query = ctx.request.query
+  query.filters = {
+    ...(query.filters || {}),
+    user: user.id
```

Comment:
Is the goal of this function to set the query to the current logged in user, or check if the query is trying to access another user? The name `is-notification-owner` sounds like the later.